crates/ruff_python_formatter/src/comments/mod.rs

```diff
@@ -514,7 +514,7 @@ mod tests {
 
     use ruff_formatter::SourceCode;
     use ruff_python_ast::{Mod, PySourceType};
-    use ruff_python_parser::{parse, AsMode, Parsed};
+    use ruff_python_parser::{parse, AsMode, Parsed, ParserOptions};
```

Comment:
I'd prefer `ParseOptions` over `ParserOptions` to align it with `FormatOptions` and because its the options passed to the `parse_*` methods (the parser is an internal abstraction)